func foobar() {
	return (x) ~ Error => {
		throw new Error()
	}
}